views/extras.tmpl

```diff
@@ -136,6 +136,15 @@
 				target="_blank"
 				rel="noopener noreferrer"
 			>dcrdata v{{.Version}}</a>
+			{{- if ne .Links.OnionV2 "" }}
+			<a
+				class="text-nowrap d-none d-md-inline-block pr-3"
+				href="{{.Links.OnionV2}}"
+				title="Hidden service URL"
+				target="_blank"
+				rel="noopener noreferrer"
+			>Onion v2</a>
```

Comment:
As a Tor user, do you think there is much value in the "v2" part of the link text?  If I were naming it I might go with "Tor host" or something like that.

Reply:
Also, can we just abandon the v2 hostname in favor of the v3 hostname?  I expect most tor versions support it these days, but I don't know for sure.

Reply:
Done - there's indeed no point in keeping the v2 hostname.